i3pystatus/spotify.py

```diff
@@ -1,105 +1,98 @@
-import math
 from i3pystatus import formatp
 from i3pystatus import IntervalModule
+from i3pystatus.core.util import TimeWrapper
+
+import gi
+gi.require_version('Playerctl', '1.0')  # nopep8
 from gi.repository import Playerctl
 
 
 class Spotify(IntervalModule):
     """
-    Gets Spotify info using playerctl
+    Gets Spotify (or any supported player) info using playerctl
 
     .. rubric:: Available formatters
 
-    * `{status}` — current status icon (paused/playing)
-    * `{length}` — total song duration (mm:ss format)
+    * `{status}` — current status icon (paused/playing/stopped)
+    * `{length}` — total song duration, uses TimeWrapper formatting, default format is `%E%l:%M:%S`
     * `{artist}` — artist
     * `{title}` — title
     * `{album}` — album
     """
 
     settings = (
         ('format', 'formatp string'),
-        ('format_not_running', 'Text to show if cmus is not running'),
+        ('format_not_running', 'Text to show if player is not running'),
         ('color', 'The color of the text'),
-        ('color_not_running', 'The color of the text, when cmus is not running'),
+        ('color_not_running', 'The color of the text, when player is not running'),
         ('status', 'Dictionary mapping status to output'),
+        ('player_name',
+            'Name of music player, use `playerctl -l` with player running to get. If None, tries to autodetect.'),
     )
 
     # default settings
     color = '#ffffff'
     color_not_running = '#ffffff'
     format = '{status} {length} {artist} - {title}'
     format_not_running = 'Not running'
-    interval = 1
     status = {
         'paused': '▷',
         'playing': '▶',
+        'stopped': '■',
     }
+    player_name = None
```

Comment:
Imho the default shouldn't change from "spotify" (#389 fixed the issue that with Spotify and other players active at the same time it's unclear which player would be chosen by i3pystatus).